src/System.Private.Xml/src/System/Xml/IApplicationResourceStreamResolver.cs

```diff
@@ -8,12 +8,12 @@
 
 namespace System.Xml
 {
-    [Obsolete("This API supports the .NET Framework infrastructure and is not intended to be used directly from your code.", true)]
+    [Obsolete("This API supports the .NET Framework infrastructure and is not intended to be used directly from your code.", false)]
```

Comment:
Why are you changing these?

Reply:
If the second attribute is true, the compiler throws an error instead of throwing a warning

Reply:
Even with a pragma?   This needs to match https://github.com/dotnet/corefx/blob/16689aa0a933517dc45465fc6edb6223222414a6/src/System.Xml.ReaderWriter/ref/System.Xml.ReaderWriter.cs#L33.

Reply:
yes pragma just works on the warning not on errors.

Reply:
No we don't want to make that change in the ref.  My point was that we shouldn't change the behavior here.  It's actually less bad to change only in the implementation.  You may need to suppress an API compat message about the attributes being different.  I couldn't find a way to getaround the Obsolete(..., true) error: it appears that's a hard block.

Reply:
api compat is not checking attribute values as I am not getting any warning or error messages for this csproj

Reply:
Interesting that the compiler has this behavior with type of.  I can use the obsolete type in a method signature if the method itself is also marked obsolete.  There doesn't appear tobe a way to use the typeof() expression at all when the type is obsolete.  That's odd.

Reply:
yeah, what about if we create a class which implements ```IApplicationResourceStreamResolver``` and then use typeof(newclass).BaseType. I am not sure if baseType wil return the interface name

Reply:
we cant inherit the obsolete type

Reply:
You can't access a member of a type in a typeof expression: it has to be statically known at compile time.  @jaredpar are you aware of anyway to circumvent CS0619 when using typeof(foo) expression when foo is a error-obsoleted type?

Reply:
@agocke may know a way here.

Reply:
FWIW I poked around the roslyn source and couldn't find a way to suppress this.  That's understandable considering its an *error* and a compiler with the ability to suppress errors would be pretty shaky.  I did find it interesting that the compiler would ignore the error so long as the caller was attributed with Obsolete, but we can't attribute a typeforward with Obsolete so... 🤷‍♂️ 

@Anipik I'm ok with you modifying implementation only to Obsolete(..., false) only for the type.